### PR TITLE
Fix keys upload limit

### PR DESCRIPTION
**Pull request by henrytao-me**
Resolves https://github.com/CovidShield/mobile/issues/192

### CODE REVIEW

File: src/services/BackendService/BackendService.ts [hunk 1]
```diff
@@ -58,7 +60,20 @@ export class BackendService implements BackendInterface {
...
+    // Ref https://github.com/CovidShield/mobile/issues/192
+    const filteredExposureKeys = Object.values(
+      _exposureKeys
+        .sort((first, second) => second.rollingStartIntervalNumber - first.rollingStartIntervalNumber)
+        .filter((value, index, array) => {
+          const previousIndex = array.findIndex(
+            lookup => lookup.rollingStartIntervalNumber === value.rollingStartIntervalNumber,
+          );
+          return previousIndex === index;
+        }),
```
Comment (honkfestival) [inline src/services/BackendService/BackendService.ts:73]: This section removes TEKs with the same rolling start number? How often does that happen?

Comment (henrytao-me) [reply src/services/BackendService/BackendService.ts:73]: I encountered once in the past 2 weeks.

Comment (honkfestival) [reply src/services/BackendService/BackendService.ts:73]: Was that potentially from changing the device date? We shouldn't need to handle that particular case on the client side; let's remove this code and see if it happens again.

Comment (henrytao-me) [reply src/services/BackendService/BackendService.ts]: Updated!

**Pull request merged**